app/views/layouts/application.html.erb

```diff
@@ -35,6 +35,10 @@
       <% elsif admin_user? %>
         <%= link_to "Dashboard", admin_dashboard_path %>
       <% end %>
+
+      <% if admin_user? %>
+        <%= link_to 'Users', admin_users_path %>
```

Comment:
To dry this up, we could move this link into the existing `if admin_user?` on line 35